trymito.io/pages/use-cases/python-training.tsx

```diff
@@ -0,0 +1,320 @@
+                            </div>
+                            <h3>
+                                Reduce time to useful automation
+                            </h3>
+                            <p>
+                                The best way to convince business analysts to commit to Python training is to start saving them time as fast as possible. 
+                            </p>
+                            <p>
+                                If your analysts leave your training without having built a useful Python automation, you&apos;re teaching them that they don&apos;t have enough time to learn Python.
+                            </p>
+                            <p>
+                                Mito let&apos;s your analysts start building useful automations in the first 10 minutes.
+                            </p>
+                        </div>
+                    </div>
+                    <div className={pageStyles.subsection}>
+                        <div className={securityStyles.security_bullet_container}>
+                            <div className={securityStyles.icon}>
+                                <Image  src={dataIconPurple} alt='icon'></Image>
+                            </div>
+                            <h3>
+                                Train in the context of your data
+                            </h3>
+                            <p>
+                                Training is most engaging when you&apos;re practicing on real business data. Because Mito hides the complexity of learning Python syntax, you don&apos;t have to oversimplify your data. 
+                            </p>
+                            <p>
+                                Instead of learning <code>total = x + y</code> skip straight to building pivot tables on your monthly returns data.
+                            </p>
+                        </div>
+                        <div className={pageStyles.subsection_second_element_mobile_spacing}>
+                            <div className={securityStyles.icon}>
+                                <Image className={securityStyles.icon} src={bugIconPurple} alt='icon'></Image>
+                            </div>
+                            <h3>
+                                Decrease errors by 8x
+                            </h3>
+                            <p>
+                                No more syntax errors or forgetting a colon. Mito generates code for you, so your analysts can focus on the business logic. 
+                            </p>
+                            <p>
+                                Help your analysts leapfrog the <a className={pageStyles.link} href='https://www.trymito.io/blog/10-mistakes-to-look-out-for-when-transitioning-from-excel-to-python' target="_blank" rel="noreferrer">“Oh my god. Why is this not working? UGGHHH!!!”</a> phase. 
```

Comment:
A little too much. Maybe `Help your analysts leapfrog the debugging-hell that is beginner programming`